src/main.cpp

```diff
@@ -267,17 +267,14 @@ static void print_device_status(std::ostream& os, const scsi::page_des& opt)
        << static_cast<unsigned int>(opt.algorithm_index) << '\n';
   }
   for (const scsi::kad& kd: scsi::read_page_kads(opt)) {
-    switch (kd.type) {
-    case scsi::kad_type::ukad:
+    if (kd.type == scsi::kad_type::ukad) {
```

Comment:
Why did you prefer if over switch here? I think switch shows more the structure and is more readable. As far I know switch would be the recommended style here.

Reply:
Compiler was printing warnings (if you compile with `-Wall` or on FreeBSD) that not all the possible values of `scsi::kad_type` are being used. I did in fact define all the possible values of KAD type that the SCSI standard allows, but stenc only shows UKAD and AKAD, and only allows UKAD to be set.

One of the benefits of the modern C++ `enum class` is you can switch them, and not have a `default` case since you can be sure you have exhaustively listed all values. But that doesn't help here, since we aren't doing anything with some of the cases. So there are some options since it's good to clean up warnings from code:

1. Remove the switch statement (I did this...)
2. Keep the switch statement, remove the unused values in the enum definition
3. Keep the unused enum values, either add a default case that does nothing, or print a label without contents (since these KAD types usually contain binary data)

What do you think?